### DISCUSSION TITLE
What are some specific examples of how Terragrunt helps with using Terraform?

### DISCUSSION THREAD

**Discussion by marijakstrazdas** [gruntwork-io/knowledge-base#257] [Help] [answered]
What are some specific examples of how Terragrunt helps with using Terraform?

**Comment by yorinasub17** [comment, marked as answer]
This [blog post](https://blog.gruntwork.io/terragrunt-how-to-keep-your-terraform-code-dry-and-maintainable-f61ae06959d8) walks through a series of examples and features of Terragrunt where it addresses some of the pain points of using Terraform directly.
